ethereum.star

```diff
@@ -60,6 +60,22 @@ def run(plan, args):
                 # GENESIS_DELAY, Default: 12
                 # This is a grace period to allow nodes and node operators time to prepare for the genesis event. The genesis event cannot occur before MIN_GENESIS_TIME. If MIN_GENESIS_ACTIVE_VALIDATOR_COUNT validators are not registered sufficiently in advance of MIN_GENESIS_TIME, then Genesis will occur GENESIS_DELAY seconds after enough validators have been registered.
                 "genesis_delay": 12,
+                "additional_preloaded_contracts": {
+                    # Deterministic deployer (used by the op-deployer).
+                    "0x4e59b44847b379578588920cA78FbF26c0B4956C": {
```

Comment:
This address seems to be already being used by the [l2-contract-setup](https://github.com/0xPolygon/kurtosis-cdk/blob/main/templates/contract-deploy/run-l2-contract-setup.sh#L83) script. When setting [deploy_l2_contracts](https://github.com/0xPolygon/kurtosis-cdk/blob/main/input_parser.star#L28) to `True`, the deployment will fail, because the check will expect no code to be present at that specific address.

Reply:
oh nice catch, I'll take a look!

Reply:
I removed the additional preloaded contracts from the ethereum package args and deployed the deterministic proxy by default on L1.